hwilib/_gui.py

```diff
@@ -303,12 +320,12 @@ def __init__(self, passphrase='', chain=Chain.MAIN):
         self.ui.enumerate_combobox.currentIndexChanged.connect(self.get_client_and_device_info)
 
     def clear_info(self):
-        self.ui.getxpub_button.setEnabled(False)
-        self.ui.signtx_button.setEnabled(False)
-        self.ui.signmsg_button.setEnabled(False)
-        self.ui.display_addr_button.setEnabled(False)
-        self.ui.getkeypool_opts_button.setEnabled(False)
-        self.ui.toggle_passphrase_button.setEnabled(False)
+        self.ui.getxpub_button.setEnabled(True)
+        self.ui.signtx_button.setEnabled(True)
+        self.ui.signmsg_button.setEnabled(True)
+        self.ui.display_addr_button.setEnabled(True)
+        self.ui.getkeypool_opts_button.setEnabled(True)
+        self.ui.toggle_passphrase_button.setEnabled(True)
```

Comment:
I don't think this is correct. We don't want users to be able use these dialogs if no device is selected yet.